src/main/java/edu/ie3/datamodel/io/extractor/Extractor.java

```diff
@@ -19,41 +19,51 @@
  * @version 0.1
  * @since 31.03.20
  */
-public class Extractor {
+public final class Extractor {
 
-  private final List<InputEntity> extractedEntities;
+  private static final Logger log = LogManager.getLogger(Extractor.class);
 
-  public Extractor(Nested nestedEntity) throws ExtractorException {
-    this.extractedEntities = extractElements(nestedEntity);
+  private Extractor() {
+    throw new IllegalStateException("Don't instantiate a utility class");
```

Comment:
Our software should not order people around :D Please change to "Utility classes cannot be instantiated"